ramalama/benchmarks/utilities.py

```
@@ -82,7 +84,13 @@ def print_bench_results(records: list[BenchmarkRecord]):
             for row in rows:
                 row.pop(field, None)
 
-    column_order = ["id", "model", "params", "backend", "ngl", "threads", "test", "t/s", "engine", "date"]
+    fields_with_defaults = {"ngl": "99", "ncmoe": "0"}
+    for key, default in fields_with_defaults.items():
```

Ruff (0.15.15)

[warning] Loop control variable `default` not used within loop body

Rename unused `default` to `_default`

(B007)